libkbfs/block_disk_store.go

```diff
@@ -268,6 +276,31 @@ func (s *blockDiskStore) hasData(id BlockID) error {
 	return err
 }
 
+var errFlushedButPresentData = errors.New("Data is flushed but present")
```

Comment:
per my pending PR, you'll probably have to change this into a type or just inline the error creation